server/jobs/example.py

```diff
@@ -1,13 +1,35 @@
 import time
+import tempfile
 
 from server import jobs
+from server.models import ExternalFile, db
+from server.utils import upload_file, encode_id
 
 @jobs.background_job
-def test_job(duration=0, should_fail=False):
+def test_job(duration=0, should_fail=False, make_file=False):
     logger = jobs.get_job_logger()
 
     logger.info('Starting...')
     time.sleep(duration)
     if should_fail:
         1/0
+    if make_file:
```

Comment:
I kind of stuck this in here so that I could test file creation in production. I can remove it once there is another place to put an example of how to use file upload.